source/tests/pt/model/test_dipole_fitting.py

```diff
@@ -0,0 +1,116 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+import itertools
+import unittest
+
+import numpy as np
+import torch
+
+from deepmd.dpmodel.fitting import DipoleFitting as DPDipoleFitting
```

Comment:
I have not tested consistency with TF yet. Is there any available fixture we can use to test cross framework consistency? probably need to export TF model weights and import to PT model.

Reply:
See #3263